tests/test_nn/test_message_passing.py

```diff
@@ -12,6 +12,18 @@ class BlockDecompositionTests(cases.DecompositionTestCase):
     cls = pykeen.nn.message_passing.BlockDecomposition
 
 
+class EfficientBasesDecompositionTestCase(cases.BasesDecompositionTestCase):
+    """Tests for efficient bases decomposition."""
+
+    cls = pykeen.nn.message_passing.EfficientBasesDecomposition
+
+
+class EfficientBlockDecompositionTestCase(cases.DecompositionTestCase):
```

Comment:
why not inherit from `BlockDecompositionTests`?

Reply:
If this one works better than the old one, I intend to delete the old ones.